lib/cronitor/monitor.rb

```diff
@@ -186,11 +187,6 @@ def fallback_ping_api_url
       "https://cronitor.io/p/#{api_key}/#{key}"
     end
 
-    def monitor_api_url
```

Comment:
No longer needed since instance and class methods now just utilize the constant.